package-system/AWSNativeSDK/README.md

```diff
@@ -0,0 +1,100 @@
+# Build New AWS Service Target in AWSNativeSDK
+Find and identify the service package required from the package directory at [aws/aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp) on GitHub.
+The package name is the suffix of the directory name for the service.
```

Comment:
nit: Add a note that several AWS services vend multiple client libraries, use the AWS C++ instructions to identify the client target(s) for the AWS Service.

Reply:
Can you clarify on this "use the AWS C++ instructions to identify the client target(s) for the AWS Service"?
Agree that some target may require multiple ones, but even with this note I don't know what is the instruction here?

Reply:
Maybe: "Find and identify the service package(s) required from the package directory at [aws/aws-sdk-cpp](https://github.com/aws/aws-sdk-cpp) on GitHub. Refer to the [AWS SDK for C++ Developer Guide](https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/welcome.html) for further help in selecting the packages by service." ?